packages/nextra-theme-docs/src/components/toc.tsx

```diff
@@ -29,31 +31,6 @@ const getEditUrl = (repository?: string, filepath?: string): string => {
   return '#'
 }
 
-const getCreateFeedbackUrl = (
-  repository?: string,
-  filepath?: string,
-  labels?: string
-): string => {
-  const repo = parseGitUrl(repository || '')
-  if (!repo) throw new Error('Invalid `docsRepositoryBase` URL!')
-  const config = useConfig()
-
-  switch (repo.type) {
-    case 'github':
-      return `https://github.com/${repo.owner}/${
-        repo.name
-      }/issues/new?title=${encodeURIComponent(
-        `Feedback for “${config.title}”`
-      )}&labels=${labels || ''}`
-    case 'gitlab':
-      return `https://gitlab.com/${repo.owner}/${repo.name}/-/blob/${
-        repo.branch || 'master'
-      }/${repo.subdir || 'pages'}${filepath}`
```

Comment:
seems like the previous GitLab issue link never worked.. it should be
https://github.com/shuding/nextra/blob/9c7cde326298887f8a60f27a3ef8f8375b300412/packages/nextra-theme-docs/src/utils/get-git-issue-url.ts#L20-L23